src/backend/distributed/replication/multi_logical_replication.c

```diff
@@ -1808,7 +1808,7 @@ TotalRelationSizeForSubscription(MultiConnection *connection, char *command)
 	{
 		char *resultString = PQgetvalue(result, 0, 0);
 
-		remoteTotalSize = pg_strtouint64(resultString, NULL, 10);
+		remoteTotalSize = strtou64(resultString, NULL, 10);
```

Comment:
```suggestion
		remoteTotalSize = SafeStringToUint64(resultString);
```

Reply:
do you know why we only have *64* versions not *32* ones in safestring?